.changeset/fast-radios-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/i18n": patch
+---
+
```

Comment:
Any specific reason for multiple changesets?

Reply:
To cover a previous PR